src/PrestaShopBundle/Form/Admin/Configure/ShopParameters/TrafficSeo/Meta/SetUpUrlType.php

```diff
@@ -52,44 +54,92 @@ class SetUpUrlType extends AbstractType
      */
     private $isHostMode;
 
+    /*
+    * @var Tools
+    */
+    private $tools;
+
     /**
      * SetUpUrlType constructor.
      *
+     * @param \Symfony\Component\Translation\TranslatorInterface $translator
+     * @param array $locales
      * @param array $canonicalUrlChoices
      * @param bool $isHtaccessFileWritable
      * @param bool $isHostMode
+     * @param Tools $tools
      */
     public function __construct(
+        TranslatorInterface $translator,
+        array $locales,
         array $canonicalUrlChoices,
         $isHtaccessFileWritable,
-        $isHostMode
+        $isHostMode,
+        $tools
     ) {
+        parent::__construct($translator, $locales);
         $this->canonicalUrlChoices = $canonicalUrlChoices;
         $this->isHtaccessFileWritable = $isHtaccessFileWritable;
         $this->isHostMode = $isHostMode;
+        $this->tools = $tools;
     }
 
     /**
      * {@inheritdoc}
      */
     public function buildForm(FormBuilderInterface $builder, array $options)
     {
+        $friendlyUrlHelp = $this->trans(
+            'Enable this option only if your server allows URL rewriting (recommended).',
+            'Admin.Shopparameters.Help'
+        );
+
+        if (!$this->tools->isModRewriteActive()) {
+            $friendlyUrlHelp .=
+                '<br/>' . $this->trans(
+                'URL rewriting (mod_rewrite) is not active on your server, or it is not possible to check your server configuration. If you want to use Friendly URLs, you must activate this mod.',
+                    'Admin.Shopparameters.Help'
+            );
+        }
+
         $builder
-            ->add('friendly_url', SwitchType::class)
-            ->add('accented_url', SwitchType::class)
+            ->add('friendly_url', SwitchType::class, [
+                'label' => $this->trans('Friendly URL', 'Admin.Global'),
+                'help' => $friendlyUrlHelp,
+            ])
+            ->add('accented_url', SwitchType::class, [
+                'label' => $this->trans('Accented URL', 'Admin.Global'),
+                'help' => $this->trans(
+                    'Enable this option if you want to allow accented characters in your friendly URLs. You should only activate this option if you are using non-latin characters ; for all the latin charsets, your SEO will be better without this option.',
```

Comment:
@JevgenijVisockij, did you decide to reunite the two translation strings into a single one? I'm okay with this option if @Julievrz is, it could be the occasion to fix a few typos, cf. the suggestion below.

Reply:
```suggestion
                    'Enable this option if you want to allow accented characters in your friendly URLs. You should only activate this option if you are using non-Latin characters; for all the Latin charsets, your SEO will be better without this option.',
```

Reply:
@LouiseBonnard 

This one confused me for a second. The reason I changed this is because in this case I copied the string from BO instead of code. Well turns out by default in english

> Enable this option if you want to allow accented characters in your friendly URLs.

 is translated to 

> Enable this option if you want to allow accented characters in your friendly URLs. You should only activate this option if you are using non-latin characters ; for all the latin charsets, your SEO will be better without this option.

So I can fix this to be like original, or I can change to reflect the english translation with changes you mentioned.

Reply:
I think it is better to keep your suggestion (1 string that combines both sentences) and apply @LouiseBonnard's suggestions. :) Thank you @JevgenijVisockij !

Reply:
Alright, will do that!